docutron/src/styles/main.css

```diff
@@ -124,8 +138,34 @@ code, pre {
 	color: #00B975;
 }
 
+.content-area h1,
+.content-area h2,
+.content-area h3,
+.content-area h4,
+.content-area h5 {
+	position: relative;
+	border-bottom: 1px solid #EDF3F1;
```

Comment:
I personally don't like this border, I think we should drop it. Feel free to ignore, it's just a personal design opinion.

Reply:
Yes, I might remove.